grid/deploy/node.py

```diff
@@ -0,0 +1,184 @@
+from grid.deploy import BaseDeployment
+from grid import utils as gr_utils
+
+import sys
+import os
+
+
+class NodeDeployment(BaseDeployment):
```

Comment:
I think this is not (and should not be) a general node deployment abstraction. It is actually a "HerokuNodeDeployment", right?

Reply:
Done! (b2843ea)